open-codegen/opengen/config/build_config.py

```diff
@@ -18,11 +20,13 @@ def __init__(self, build_dir=None):
             A new instance of BuildConfiguration
 
         """
+        random_string = ''.join(random.choice(string.ascii_letters) for i in range(20))
+
         self.__target_system = 'default'
         self.__build_mode = 'release'
-        self.__cost_function_name = 'phi'
-        self.__grad_cost_function_name = 'grad_phi'
-        self.__constraint_penalty_function = 'constraints_penalty'
+        self.__cost_function_name = 'phi_' + random_string
+        self.__grad_cost_function_name = 'grad_phi_' + random_string
+        self.__constraint_penalty_function = 'constraints_penalty_' + random_string
```

Comment:
I will make a commit to write these in the YAML file of the auto-generated project.

Reply:
Not sure what is the issue?

Reply:
I have added this